bundles/config/org.eclipse.smarthome.config.discovery/src/main/java/org/eclipse/smarthome/config/discovery/DiscoveryResultBuilder.java

```diff
@@ -128,13 +129,24 @@ public DiscoveryResultBuilder withTTL(long ttl) {
     }
 
     /**
+     * Sets the class name of the {@link DiscoveryService} that has discovered this {@link DiscoveryResult}.
+     *
+     * @param service class name of the {@link DiscoveryService}
+     * @return the updated builder
+     */
+    public DiscoveryResultBuilder withDiscoveryService(String service) {
```

Comment:
Oh you don't need to adapt gazillions of bindings. It is perfectly valid to just not set the discoveredBy... And: it is already handled in the ABstractDiscoveryService, no worries. I can even remove it from the builder completely, not needed.